daemon/container.go

```diff
@@ -1048,6 +1049,57 @@ func (container *Container) HostConfig() *runconfig.HostConfig {
 	return res
 }
 
+func (c *Container) WriteAs(container *Container, config *ContainersConfig, newC *types.Container) {
+	c.Lock()
+	defer c.Unlock()
+	newC.Image = container.Config.Image
+	newC.HostConfig.NetworkMode = string(c.hostConfig.NetworkMode)
+	if len(container.Args) > 0 {
+		args := []string{}
+		for _, arg := range container.Args {
+			if strings.Contains(arg, " ") {
+				args = append(args, fmt.Sprintf("'%s'", arg))
```

Comment:
Perhaps better to keep the old `'%s'`, it's a non-related change anyway, I shouldn't have mentioned it.